.gitbook/5-oracles/1-operate-oracle-nodes/2-oracle-intialization.md

```diff
@@ -0,0 +1,134 @@
+# Initialization
+
+#### Before run the oracle, you should initialize the oracle configuration.
+
+## Preparations
+
+Before initializing the oracle, you should install the oracle described
+in [oracle-installation](./1-oracle-installation.md).
+After install the oracle, create an empty directory on your host, to be mounted as the `/home_mnt` directory in the
+enclave.
+
+```bash
+mkdir <directory-you-want>/oracle
+```
+
+After that, if you want to run the `oracled` with Docker as described in
+the [oracle-installation](./1-oracle-installation.md),
+it is recommended to create an environment variable that you can execute the Docker container easily.
+
+```bash
+export ORACLE_CMD="docker run --rm \
+  --device /dev/sgx_enclave \
+  --device /dev/sgx_provision \
+  -v $(pwd)/oracle:/oracle ghcr.io/medibloc/panacea-oracle:latest \
+  ego run /usr/bin/oracled"
+```
+
+## Command Line of Initialization
+
+```bash
+$DOCKER_CMD init --home $HOME/.oracle 
```

Comment:
And the path for home probably should be `/home_mnt/.oracle`, right?.

Reply:
Right, I missed it.